core-plugin/testSrc/com/google/gct/idea/debugger/CloudDebugProcessTest.java

```diff
@@ -72,7 +74,7 @@ public void testRegisterAdditionalActions_close() {
 
         assertEquals(3, leftToolbar.getChildrenCount());
         assertEquals(action0, leftToolbar.getChildActionsOrStubs()[0]);
-        assertEquals(action2, leftToolbar.getChildActionsOrStubs()[1]);
+        assertEquals(action3, leftToolbar.getChildActionsOrStubs()[1]);
```

Comment:
I think this test is enough for now. If I read it right, this covers the case where the URL is empty. Later we'll make the uRL non-empty, this test will fail, and we'll update it. We don't expect to have both empty and non-empty caes at the same time. Is that correct?